_sources/rust-2019-rethinking-the-team-structure.md

```diff
@@ -0,0 +1,39 @@
+
+I don't think it's going to carry us to the next level. It believe constrains us with regards to bandwidth _and_ participation. A couple of ideas arouund that subject:
+
+## Single leaders
+
+I think it's time to move beyond single leaders. It's odd that we build our teams towards independence, but the teams themselves are hierarchical. The community team is currently experimenting with floating roles and experiences look good. It has multiple advantages: it shifts away responsible from full-time Rust employees, while still being flexible enough to have hobbyists choose their own pace and effort. It also makes the team resilient against sudden changes. I'd like to specifically mention [Ashley Williams](TODO: Twitter link) for starting the change towards that model.
+
+## Defined goals, roles and responsibilities better
+
+For most teams and W, the written down description and mission doesn't go beyond their name. This may not sound like a huge problem, but it makes it hard to evaluate progress. While we're doing public roadmapping on a project level, public roadmapping on a team level might make sense.
+
+Also, not all teams are the same. While many of the technical teams have a clesarly defined role, some teams like community, mods and core have more flexible roles, they play different coordination roles. From the perspective of the community team, this job has become harder as the project grows.
+
+From my point of view, this does for example stem from the fact that we haven't clearly defined what we want to be kept informed about and what not or what other teams want to be informed about from us.
+
+I think noting down these connections and expectations for all teams would be important.
+
+## Better input for specialists
+
+The community team has set up (but, admittedly, underused it) a system of "associates". Those take no team members and generally have no interest in joining meetings or following along. They register interest for certain subjects and will be queried when the subject arises.
+
+This deal is to the benefit of both sides: community members are not feeling ignored on a subject they are good at, but the team also doesn't need to carry the weight of keeping a full team member around that has no interest or no capacity to follow team business in full. This is a perfect position for hobbyists.
+
+## Team setup policies
+
+How does one join a team? What's the base expectation? How to you propose a new team? How are working groups set up? All those questions are not clearly answered to our community currently.
+
+This has been an ongoing problem since Rust 1.0, when the teams were announced out of the blue. That's a ton of work we've been pushing off since then.
+
+The goal of this should not be to save each other work and give visibilty, not introduce more friction.
```

Comment:
nobody intentionally introduces friction. i think thats not valid criticism.

Reply:
Right, I'll replace that by something else.